lib/api/asset/v1/types.ts

```diff
@@ -18,7 +18,7 @@ export interface Asset {
   areaId: string;
   assetId: string;
   assetType: AssetType;
-  rentGroup: RentGroup | null;
+  rentGroup: RentGroup | undefined | null;
```

Comment:
Is this equivalent to `rentGroup?: RentGroup | null;` ?

Reply:
Yeah it's the same thing. Thinking it over I'll just change it to be `rentGroup?` instead.